src/KeyvaultHelperFunctions.ps1

```diff
@@ -109,7 +109,7 @@ DeleteAndPurgeSecretFromKeyVault -keyVaultName $keyVaultName -secretKey $name -l
 #>
 function DeleteAndPurgeSecretFromKeyVault([string]$keyVaultName, [string]$secretKey, $logger) {
 	try {
-		Add-KeyVaultRoleAssignment -keyVaultName $keyVaultName -logger $logger;
+		# Add-KeyVaultRoleAssignment -keyVaultName $keyVaultName -logger $logger;
```

Comment:
Remove the line if not needed

Reply:
Tagging @M2skills for more infor. Should we remove it?

Reply:
yes, remove this as we are providing access at the beginning of the main function

Reply:
removed